caffe2/operators/upsample_op.cu

```diff
@@ -61,42 +60,21 @@ __global__ void UpsampleBilinearKernel(
     const float w1lambda = w1r - w1;
     const float w0lambda = 1.f - w1lambda;
 
-    Y[index] =
-        (h0lambda *
-             (w0lambda *
-                  X[idx(
-                      n, num_channels, c, input_height, input_width, h1, w1)] +
-              w1lambda *
-                  X[idx(
-                      n,
-                      num_channels,
-                      c,
-                      input_height,
-                      input_width,
-                      h1,
-                      w1 + w1p)]) +
-         h1lambda *
-             (w0lambda *
-                  X[idx(
-                      n,
-                      num_channels,
-                      c,
-                      input_height,
-                      input_width,
-                      h1 + h1p,
-                      w1)] +
-              w1lambda *
-                  X[idx(
-                      n,
-                      num_channels,
-                      c,
-                      input_height,
-                      input_width,
-                      h1 + h1p,
-                      w1 + w1p)]));
+    for (int n = 0; n < num_batch; n++){
```

Comment:
Not sure if here we can have some better solution. But it's fine to keep it as this now.